lib/manager/npm/post-update/npm.ts

```diff
@@ -111,9 +111,11 @@ export async function generateLockFile(
     await exec(commands, execOptions);
 
     // massage to shrinkwrap if necessary
+    const localDir = GlobalConfig.get('localDir');
+    const subDir = cwd === localDir ? '' : cwd.replace(localDir, '');
```

Comment:
@rarkins i think we should move this a level up, so we don't need to recompute it for every npm manager. 🤔

@olegkrivtsov i think we have a getSilbling like function for getting beside package file.

Reply:
I've used `findLocalSiblingOrParent` as you suggested.